src/register/index.js

```diff
@@ -59,7 +59,7 @@
         window.location.replace('/login');
       }
     } catch (error) {
-      document.querySelector(`label[for=submit]`).style.display = 'flex';
+      console.log(error.message);
```

Comment:
Hmm yeah, only thing up for debate I guess could be this one. I dont mind having a console.log when an error happens. Might help us debug a situation if a user has a problem, =)